armory/art_experimental/attacks/poison_loader_obj_det.py

```diff
@@ -0,0 +1,33 @@
+from armory.art_experimental.attacks.poison_loader_dlbd import poison_loader_dlbd
+from armory.utils import config_loading
+
+bad_det_attacks = [
+    "BadDetRegionalMisclassificationAttack",
+    "BadDetGlobalMisclassificationAttack",
+    "BadDetObjectGenerationAttack",
+    "BadDetObjectDisappearanceAttack",
```

Comment:
Should these be listed in https://github.com/twosixlabs/armory/blob/develop/docs/poisoning_object_detection.md#configuration-files?
Not a huge deal since error should inform user of options.